spec/include_directive/parser_spec.rb

```diff
@@ -28,6 +28,19 @@
     expect(hash).to eq expected
   end
 
+  it 'treats spaces as part of the resource name' do
+    str = 'friends, comments.author , posts.author,posts. comments.author'
+    hash = JSONAPI::IncludeDirective::Parser.parse_include_args(str)
+    expected = {
+      friends: {},
+      ' comments': { 'author ': {} },
```

Comment:
Ruby 2.1 complains about this syntax. Let's change it to `' comments' => { ... }`?